### PR TITLE
[Bitget] add Bitget compatibility

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
Changes in exchanges are required to be able to define a custom ccxt connector (to get Bitget timeframes)
Used in https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/657

### CODE REVIEW

File: tests_additional/real_exchanges/test_bitget.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+from tests_additional.real_exchanges.real_exchange_tester import RealExchangeTester
+from tests_additional.real_exchanges import get_exchange_manager
+# required to catch async loop context exceptions
+from tests import event_loop
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+
+class TestBitgetRealExchangeTester(RealExchangeTester):
```
Comment (Herklos) [inline tests_additional/real_exchanges/test_bitget.py:31]: 👍

File: octobot_trading/exchanges/implementations/spot_ccxt_exchange.py [hunk 2]
```diff
@@ -33,10 +33,11 @@ class SpotCCXTExchange(exchanges_types.SpotExchange):
     ORDER_NON_EMPTY_FIELDS = [ecoc.ID.value, ecoc.TIMESTAMP.value, ecoc.SYMBOL.value, ecoc.TYPE.value,
                               ecoc.SIDE.value, ecoc.PRICE.value, ecoc.AMOUNT.value, ecoc.STATUS.value]
     ORDER_REQUIRED_FIELDS = ORDER_NON_EMPTY_FIELDS + [ecoc.REMAINING.value]
+    CONNECTOR_CLASS = exchange_connectors.CCXTExchange
```
Comment (Herklos) [inline octobot_trading/exchanges/implementations/spot_ccxt_exchange.py:36]: 👍

**Review by Herklos — APPROVED** [review]

**Comment by GuillaumeDSM** [comment]
we merge  ?

File: .github/workflows/cron.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        exchange: [ ascendex, binance, bitfinex, bithumb, bitstamp, bittrex, bybit, coinbasepro, crex24, ftx, gateio, hollaex, hitbtc, huobi, huobipro, kraken, kucoin, okcoin, okx, poloniex, upbit, wavesexchange ]
+        exchange: [ ascendex, binance, bitfinex, bithumb, bitstamp, bittrex, bybit, coinbasepro, crex24, ftx, gateio, hollaex, hitbtc, huobi, huobipro, kraken, kucoin, okcoin, okx, poloniex, upbit, wavesexchange, bitget ]
```
Comment (Herklos) [inline .github/workflows/cron.yml:12]: 🎉

**Review by Herklos — APPROVED** [review]
👍

**Pull request merged**